docs/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: nf-co2footprint
+description: Nextflow plugin for CO2 footprint estimations
```

Comment:
```suggestion
description: Nextflow plugin for CO<sup>2</sup> footprint estimations
```

Reply:
not sure if this works

Reply:
Isn't it better to have plain text in the description?